### DISCUSSION TITLE
Azure.VM.NICName - 80 char limit

### DISCUSSION THREAD

**Discussion by paulmcclbg** [Azure/PSRule.Rules.Azure#2641] [Q&A]
Where did the 80 char limit come from? We are seeing NICs with names > 80 and therefore the rule is failing.

**Comment by BernieWhite** [comment]
See rule docs link: https://learn.microsoft.com/en-au/azure/azure-resource-manager/management/resource-name-rules#microsoftnetwork

Are the nics related to private link or endpoints?
